fuse-products/src/main/java/software/tnb/product/cxf/OpenshiftCxfQuarkus.java

```diff
@@ -0,0 +1,18 @@
+package software.tnb.product.cxf;
+
+import software.tnb.product.Product;
+import software.tnb.product.application.App;
+import software.tnb.product.cq.OpenshiftCamelQuarkus;
+import software.tnb.product.cxf.app.PlainQuarkusApp;
+import software.tnb.product.integration.builder.AbstractIntegrationBuilder;
+
+import com.google.auto.service.AutoService;
+
+@AutoService(Product.class)
+public class OpenshiftCxfQuarkus extends OpenshiftCamelQuarkus {
```

Comment:
I'd say this should `extend OpenshiftProduct` directly, as it has nothing to do with CQ

Reply:
In the name of logic I also suggest at least to change logger class in OpenshiftQuarkusApp.java to OpenshiftQuarkusApp. It will not be perfect as it is in cq package but at least it wont be misleading in logs. WDYT?